### DISCUSSION TITLE
A lot of requests come from my router ip (they are not from it)

### DISCUSSION THREAD

**Discussion by CtrlValCanc** [AdguardTeam/AdGuardHome#7448] [Q&A]
![immagine](https://github.com/user-attachments/assets/0b6d0e82-8ff8-437e-849e-07aaaefc7153)
As you can see, I have ~50% of requests come from my router local ip (fritzbox 7530). If I look at the requests, they are not from my router (I can see EA, Amazon, Spotify, Github... stuff I'm using right now on my pc). But some requests are from my pc ip for example, so it's not like ALL my requests come from router.
I tried searching but I couldn't find a solution.
Thank you.

**Comment by Dynamic5912** [comment]
Guest network?

I believe the guest network routes traffic via the router DNS so it makes request look like they are coming from the router

**Reply by CtrlValCanc** [reply]
Thank you for your answer. I do not have guest network on.
For example I see my pc IP, some of the requests are from my pc IP, some from my router IP.

**Reply by Dynamic5912** [reply]
What's the dns settings on your router set to?

**Comment by alexsannikov** [comment]
Probably, all the clients use your router IP as a DNS, and router itself behaves like a proxy and sends all the clients requests to AdGuard Home. In this case real requests IPs are hided by the router IP.

**Comment by fvdm** [comment]
Maybe it helps to enable **EDNS0** in the FritzBox > Account Information > DNS Server > _Transmit the MAC address and the name of the querying network device_
